drivers/rtc/rtc_smartbond.c

```diff
@@ -0,0 +1,635 @@
+/*
+ * Copyright (c) 2023 Renesas Electronics Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <errno.h>
+#include <zephyr/device.h>
+#include <zephyr/kernel.h>
+#include <zephyr/irq.h>
+#include <zephyr/sys/util.h>
+#include <zephyr/devicetree.h>
+#include <zephyr/drivers/rtc.h>
+#include <zephyr/drivers/clock_control.h>
+#include <zephyr/drivers/clock_control/smartbond_clock_control.h>
+#include <DA1469xAB.h>
+#include <da1469x_config.h>
+#include <da1469x_pdc.h>
+
+
+#define LOG_LEVEL CONFIG_RTC_LOG_LEVEL
+#include <zephyr/logging/log.h>
+
+LOG_MODULE_REGISTER(rtc_smartbond);
+
+#define DT_DRV_COMPAT  renesas_smartbond_rtc
+
+#define IRQN       DT_INST_IRQN(0)
+#define IRQ_PRIO   DT_INST_IRQ(0, priority)
```

Comment:
I would either namespace these macros, or use them directly in the `IRQ_CONNECT` macro later in the file to avoid naming conflicts, something like `SMARTBOND_IRQN` etc.

Reply:
Fixed